crates/aptos/e2e/main.py

```diff
@@ -134,12 +145,23 @@ def run_tests(run_helper):
     test_move_run(run_helper)
     test_move_view(run_helper)
 
-    # WARNING: This has to stay at the end, else key will get rotated
-    test_account_rotate_key(run_helper)
+    # Run stake subcommand group tests.
+    test_stake_initialize_stake_owner(run_helper)
+    test_stake_add_stake(run_helper)
+    test_stake_withdraw_stake_before_unlock(run_helper)
+    test_stake_unlock_stake(run_helper)
+    test_stake_withdraw_stake_after_unlock(run_helper)
+    test_stake_increase_lockup(run_helper)
+    test_stake_set_operator(run_helper)
+    test_stake_set_voter(run_helper)
+    test_stake_create_staking_contract(run_helper)
 
     # Run node subcommand group tests.
     test_node_show_validator_set(run_helper)
 
+    # WARNING: This has to stay at the end, else key will get rotated
+    test_account_rotate_key(run_helper)
```

Comment:
Kinda unrealted, but you could always rotate this back in the test if you don't want to have to worry about it failing because this isn't in the correct order

Reply:
Ahh good point!

Reply:
also maybe you should note somewhere for yourself that these have to be run in this specific order, because initially I thought they were standalone test cases but they rely on the state from the previous test. Or just put all of them into one big function and then make that the test case?

Don't know if it matters, but could save yourself/someone reading this a headache in the future if they ever move it around haha

Reply:
Didn't realize auto-merge was enabled my bad lol

Reply:
@xbtmatt haha no worry! I can follow up with a new PR on this